styles.css

```diff
@@ -0,0 +1,108 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+html {
+  font-family: sans-serif, 'Poppinsp';
+}
+
+body {
+  background-color: #e5e5e5;
+  height: 918px;
+  width: 375px;
+  margin: 0;
+}
+
+header {
+  background-color: white;
+}
+
+main {
+  background-image: url("images/header-shapes\ mobile.svg"), url("images/header-background-mobile\(4\).svg");
+  height: 918px;
+  width: 375px;
+  background-repeat: no-repeat;
+  padding-top: 120px;
+}
+
+.navigation {
+  text-decoration: none;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Kindly consider aligning your logo and hamburger as in the Figma design because we must stick to the figma design and also aligning these will look good aesthetically.
your design
![qw](https://user-images.githubusercontent.com/96683843/213263698-1ee5a6c4-0680-4fa4-8dbf-b7969d3ef4b0.jpg)

figma design
![qwer](https://user-images.githubusercontent.com/96683843/213263723-5495e8b4-4d78-4ba2-8d86-e7915beeca73.jpg)